bin/ganga

```diff
@@ -55,17 +56,9 @@ def log(level, err):
         sys.stderr.write('(consider --debug option for more information)\n')
 
 try:
-    # Process options given at command line and in configuration file(s)
-    # Perform environment setup and bootstrap
-    import Ganga.Runtime
-    Ganga.Runtime._prog = Ganga.Runtime.GangaProgram()
-    Ganga.Runtime._prog.parseOptions()
-    Ganga.Runtime._prog.configure()
-    Ganga.Runtime._prog.initEnvironment()
-    Ganga.Runtime._prog.bootstrap(Ganga.Runtime._prog.interactive)
-    Ganga.Runtime._prog.new_user_wizard()
     # Import GPI and run Ganga
     from Ganga.GPI import *
+    setupGanga()
     Ganga.Runtime._prog.run()
```

Comment:
Hmm, looks like the `standardSetup` doesn't seem to take correctly and the test is over-compensating for this by inserting ganga into the python path. I'll see if I can make some changes to the standard startup script to fix this.
(I think we should __really__ be using the `__file__` property rather than the argv to determine where this file is...)